addons/mail/static/src/core/common/attachment_view.xml

```diff
@@ -4,7 +4,7 @@
     <t t-name="mail.AttachmentView">
         <div t-if="state.thread.attachmentsInWebClientView.length > 0" class="o-mail-Attachment">
             <t t-if="state.thread.mainAttachment">
-                <h3 t-if="!state.thread.mainAttachment.isPdf" class="mt0 mb8 ps-2 text-muted float-end"><t t-esc="state.thread.mainAttachment.filename"/></h3>
+                <h3 t-if="!state.thread.mainAttachment.isPdf" class="mt0 mb8 ps-2 text-muted d-flex justify-content-center"> <div> <t t-esc="state.thread.mainAttachment.filename"/> </div> </h3>
```

Comment:
Remove the extra spaces between elements; even though it's trimmed when inserted into the DOM, the `textContent` will still contain the spaces, which is annoying if you want to do assertions on it in testing, for example